konduit-serving-api/pom.xml

```diff
@@ -66,16 +66,16 @@
                     <groupId>junit</groupId>
                     <artifactId>junit</artifactId>
                 </dependency>
-                <dependency>
-                    <groupId>com.googlecode.json-simple</groupId>
-                    <artifactId>json-simple</artifactId>
-                    <version>1.1</version>
-                </dependency>
             </dependencies>
         </profile>
     </profiles>
 
     <dependencies>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
```

Comment:
We want json-simple gone. No reason for it to exist on the classpath. See: https://github.com/KonduitAI/konduit-serving/pull/62